docs/UserGuide.md

```diff
@@ -110,27 +112,27 @@ Tags: FallRisk, Diabetes
 You now have 1 patient(s) in your address book.
 ```
 
-### Listing all patients : `list`
+### Listing patients : `list`
+
+Displays a list of patients in NAB. 
 
-Displays a list of all registered patients.
+By using list with optional parameters, you will be able to list all patients.
 
-Format: `list`, `list [w\WARD] [t\TAG]...`
+Optionally, you can filter by ward or tags to list only patients in a specific ward or with specific tags.
```

Comment:
Maybe we avoid using 'you' for user guide?

Reply:
i think part of the rubrics is to use you-language to enhance engagement